### DISCUSSION TITLE
I have not received any email confirming whether my ID has been sent for review.

### DISCUSSION THREAD

**Discussion by sivoleck** [community/community#128894] [GitHub Education] [answered]
### Hi everyone,

When I did it for the first time I got an email saying that I had to personalize my account so that it would have a better chance of being approved, it said that when I did it I should log out and send it again.
That's what I did, but this time I didn't get any confirmation email to confirm that it had been sent correctly.
and I wondered whether it was normal or not.

**Comment by davevad93** [comment]
Hi @sivoleck ,

After each request GitHub should send a confirmation email about the status of your request, but I've heard other people experiencing the same thing. One question though, have you tried going to the application [page](https://education.github.com/discount_requests/application) and taking a look if there's any information there? Normally, on the right side of the page there is a record of the requests made with their status (`Rejected` or `Approved`) and all the info about it.

**Reply by sivoleck** [reply]
I can't see anything on the right
 side, just my profile and these options, and below I found this: ![Captura de pantalla 2024-06-17 230917](https://github.com/community/community/assets/92183378/fbc23977-7965-453c-a9c1-3d76a09b9ba5)
![Captura de pantalla 2024-06-17 231029](https://github.com/community/community/assets/92183378/453b4731-4412-452e-a519-c0691516ae34)

**Reply by davevad93** [reply, marked as answer]
The first picture you have shown says that you are already verified on GitHub Education since June 15. That means you are probably approved and you have to wait a while for your benefits to become effective. As you can see I have the same and my benefits are fully available since December.

![0](https://github.com/community/community/assets/94308234/726c667a-be78-4a66-b1a6-111645320f2e)

Usually when the benefits become available a `PRO` badge shows up in the profile.

![0](https://github.com/community/community/assets/94308234/290e1f7a-719b-4b1a-9332-8d2c279c627e)

Be patient and wait a few days as you'll surely have access to the Student Developer Pack, sometimes it can take up to 1 or 2 weeks but usually it takes only a few days.

**Reply by sivoleck** [reply]
OK, OK
thank you very much have a nice afternoon or evening  :)

**Reply by davevad93** [reply]
You're welcome @sivoleck , if you have any issues with that and after a couple of weeks or so you still haven't been able to access your benefits, I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and describe your issue. Have a nice evening too!
